onnxruntime/core/graph/graph.cc

```diff
@@ -3813,6 +3823,63 @@ bool Graph::GetOrtValueInitializer(const std::string& name, OrtValue& value, boo
   return false;
 }
 
+Status Graph::LoadExternalInitializerAsOrtValue(const std::string& name, OrtValue& value) {
+  auto tensor_proto_iter = name_to_initial_tensor_.find(name);
+  ORT_RETURN_IF(tensor_proto_iter == name_to_initial_tensor_.end(), "Cannot load unknown initializer named '",
+                name, "'.");
+  const ONNX_NAMESPACE::TensorProto& tensor_proto = *tensor_proto_iter->second;
+
+  auto ext_info_iter = ext_initializers_.find(name);
```

Comment:
>ext_initializers_ [](http://example.com/codeflow?start=23&length=17)

TODO: We have lots of structures to keep in sync. Need to cut down on it.

Also, do we want to check ReplaceInitialzier() functionality?